### ISSUE TITLE
How to implement 92.9% OA?

### ISSUE THREAD

**Issue by 357998846** [WangYueFt/dgcnn#65]
Hello, the OA of modelnet40 is 91.7% or 92.2%? 
How is 92.9% in the paper realistic?
My own recurrence is only 91.7%.
How to implement 92.9% OA?
Thank you very much.

**Comment by dogyoonlee** [comment]
hello, I have a question about accuracy

what is OA in code?
I mean, we can show the validation accuracy and validation accuracy with vote during training.

and we can evaluate the saved model through the test_models.py

However, the evaluated accuracy from test_models.py is different to validation accuracy or vote accuracy during training.

which one is the OA, the final results?

**Comment by guochengqian** [comment]
I confirm it is easy to achieve the 92.9 OA using their code (results reported using the best model). 
Exp show that you can achieve even better results (like 93.1) using the same number epochs as KPConv (400 epochs) instead of 250 epochs.

**Comment by sheshap** [comment]
I got 92.4%

**Comment by sunhufei** [comment]
Train 26, loss: 3.676545, train acc: 0.075407, train avg acc: 0.030953
Test 26, loss: 3.640235, test acc: 0.042139, test avg acc: 0.026000
Train 27, loss: 3.671733, train acc: 0.072358, train avg acc: 0.030758
Test 27, loss: 3.637559, test acc: 0.044976, test avg acc: 0.027750
Train 28, loss: 3.675745, train acc: 0.073272, train avg acc: 0.031713
Test 28, loss: 3.636188, test acc: 0.068071, test avg acc: 0.042000
Train 29, loss: 3.691305, train acc: 0.071545, train avg acc: 0.030454

**Comment by sunhufei** [comment]
Why is the accuracy rate so low? #81
